pkg/multus/multus.go

```diff
@@ -812,51 +930,26 @@ func CmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient *k8s.ClientInfo) er
 		defer netns.Close()
 	}
 
-	k8sArgs, err := k8s.GetK8sArgs(args)
-	if err != nil {
-		return cmdErr(nil, "error getting k8s args: %v", err)
-	}
-
-	if in.ReadinessIndicatorFile != "" {
-		err := wait.PollImmediate(pollDuration, pollTimeout, func() (bool, error) {
-			_, err := os.Stat(in.ReadinessIndicatorFile)
-			return err == nil, nil
-		})
-		if err != nil {
-			return cmdErr(k8sArgs, "PollImmediate error waiting for ReadinessIndicatorFile (on del): %v", err)
-		}
-	}
-
-	kubeClient, err = k8s.GetK8sClient(in.Kubeconfig, kubeClient)
-	if err != nil {
-		return cmdErr(nil, "error getting k8s client: %v", err)
-	}
-
-	pod, err := getPod(kubeClient, k8sArgs, true)
-	if err != nil {
-		return err
-	}
```

Comment:
Why do you remove these code from here?

Reply:
I think you're right; I accidentally delete this for `CNI_DEL`. Will put it back.